components/compile_shader.sh

```
@@ -0,0 +1,5 @@
+which qsb > /dev/null
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)